projects/go-tester/src/app/components/test-page-3/test-page-3.component.ts

```diff
@@ -53,6 +75,10 @@ export class TestPage3Component {
       if (input in this.form.controls) {
         this.form.controls[input].setErrors(errors);
       }
+
+      if (this.validationSelectControl.value.name !== 'Ron') {
+        this.validationSelectControl.setErrors(errorResponse['validationSelectControl']);
```

Comment:
Again, I know this is tester, but this is not quite working as I would expect. It looks like it only shows the error on form submit. Shouldn't it show this error until it's valid?

Reply:
It depends how we want it.. Since this is only a tester, I don't think I'm gonna make this change

Reply:
This would rely on us handling the validations on the frontend. I think we can do that for things like `required`, but I'm not sure if we want to make assumptions about validation things.